### PR TITLE
Added argument type for "java.nio.file.Path"

### PR DESCRIPTION
**Pull request by jstuyts**
The translations for German and Russian need to be checked. PR for #58.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,63 +14,68 @@
...
-                </plugin>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-gpg-plugin</artifactId>
-                    <version>1.5</version>
-                </plugin>
-            </plugins>
-        </pluginManagement>
-    </build>
+	<dependencyManagement>
```
Comment (tatsuhiro-t) [inline pom.xml]: It looks like the indentation of the modified region is broken a bit.

Comment (jstuyts) [inline pom.xml]: Indeed it is. I will fix it ASAP.

**Comment by tatsuhiro-t** [comment]
I got the following error with `mvn site`

> [ERROR] Failed to execute goal on project argparse4j-java7: Could not resolve dependencies for project net.sourceforge.argparse4j:argparse4j-java7:jar:0.8.0-SNAPSHOT: Could not find artifact net.sourceforge.argparse4j:argparse4j:jar:0.8.0-SNAPSHOT -> [Help 1]

Are there some missing dependencies?

**Comment by jstuyts** [comment]
No, I don't think so. Maven simply does not build the main module when you execute goal `site`, unless there is a reporting plug-in that depends on compiled code. Run a `mvn install` in the root first before you run `mvn site`.

See: https://stackoverflow.com/questions/7772556/maven-fails-to-resolve-dependency#7926847

**Comment by tatsuhiro-t** [comment]
`mvn package site` works for me.  Thank you!

**Pull request merged**